packages/core/types/index.js

```diff
@@ -1586,6 +1586,7 @@ export type RuntimeAsset = {|
   +dependency?: Dependency,
   +isEntry?: boolean,
   +env?: EnvironmentOptions,
+  +moveToSeparateBundle?: boolean,
```

Comment:
Maybe we should make this match dependencies? That would mean using `priority: 'parallel'` or `priority: 'async'` here. What do you think?

Reply:
This sounds reasonable to me. Are we happy to exclude `async` though as that would be a different implementation and I'm not sure what the use-case would be. Could always add later if desired?

Reply:
@devongovett I updated the API to use `priority` instead of `moveToSeparateBundle`. However, I haven't implemented `async`.